Gruntfile.js

```diff
@@ -153,6 +153,7 @@ module.exports = function (grunt) {
                 '/' + staticDir + '/geo.ext.min.js',
                 // '/' + rootStaticDir + '/libs.min.js', // libs included in jade template
                 '/' + staticDir + '/jquery.gridster.js',
+                '/' + staticDir + '/jquery-ui.min.js',
```

Comment:
I see, thanks @mbertrand @jbeezley.

I am planning to review this soon but it would be great if @mgrauer can review it as well.